mixins/request-download-file/index.js

```diff
@@ -15,27 +16,9 @@ export default {
 
       const fileName = propOr('', 'name', downloadInfo)
 
-      const requestUrl = `${process.env.portal_api}/download?key=${filePath}`
-      this.$axios.$get(requestUrl).then((response) => {
-        this._downloadFile(fileName, response)
+      this.$axios.$post(process.env.zipit_api_host, payload).then((response) => {
```

Comment:
This may be missing some commits? `payload` is not defined, and the `regexp` work that you were doing in the last PR is not here (might not be needed now though, so I'm not sure.)

Reply:
good eye. commits problems, definitely. Let me straighten that out